plugins/wordpress/capture-eye/README.md

```diff
@@ -0,0 +1,10 @@
+# Capture Eye WordPress Plugin
+
+A WordPress plugin used to add the Capture Eye script to the front-end head and add the widget in Elementor.
+
```

Comment:
Maybe there could be some installation instructions for people not familiar with wordpress plugins? Such as the steps of copying files to `wp-content/plugins/` and clicking activate plugin.